tasks/internal/process-file-updates-task/process-file-updates-task.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: process-file-updates-task
   labels:
-    app.kubernetes.io/version: "0.1.1"
+    app.kubernetes.io/version: "0.1.0"
```

Comment:
This is wrong - it is lowering the version (I would also go to 1.0.0 or at least 0.2.0.. making it idempotent is a big thing)

Reply:
OK. Updated! Thanks